src/Controller/AdminController.php

```diff
@@ -18,4 +26,41 @@ public function index(): Response
     {
         return $this->render('admin/index.html.twig');
     }
+
+    /**
+     * @Route("/add", name="add")
```

Comment:
route should be in french and be more self-explanatory (what are we adding here?)